tests/Stampie/Tests/Mailer/MailGunTest.php

```diff
@@ -2,120 +2,100 @@
 
 namespace Stampie\Tests\Mailer;
 
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use Http\Client\HttpClient;
 use Stampie\Mailer\MailGun;
+use Stampie\Tests\BaseMailerTest;
 
 /**
  * @author Henrik Bjornskov <henrik@bjrnskov.dk>
  */
-class MailGunTest extends \Stampie\Tests\BaseMailerTest
+class MailGunTest extends BaseMailerTest
 {
     const SERVER_TOKEN = 'henrik.bjrnskov.dk:myCustomKey';
 
+    /**
+     * @var MailGun
+     */
+    private $mailer;
+
+    /**
+     * @var HttpClient|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $httpClient;
```

Comment:
why not using the `$adapter` property you added in the parent ?

Reply:
The idea is to remove the adapter property from the parent and keep only helper methods in the parent class. Having an HttpClient is an implementation detail.